main/precompiled/pre-ecMul.zkasm

```diff
@@ -1,18 +1,18 @@
 /**
  * @link [https://www.evm.codes/precompiled#0x07?fork=berlin]
  * @zk-counters
- *  - dynamic steps: 130000
+ *  - dynamic steps: 175000
  *  - dynamic arith: 20000
- *  - dynamic binary: 4000
+ *  - dynamic binary: 20000
  * @process-precompiled
  *  - stack input: [x1, y1, s]
  *  - stack output: [x, y]
  */
 funcEcMul:
 
-    %MAX_CNT_BINARY - CNT_BINARY - 4000     :JMPN(outOfCountersBinary)
+    %MAX_CNT_BINARY - CNT_BINARY - 20000     :JMPN(outOfCountersBinary)
     %MAX_CNT_ARITH - CNT_ARITH - 20000      :JMPN(outOfCountersArith)
-    %MAX_CNT_STEPS - STEP - 130000          :JMPN(outOfCountersStep)
+    %MAX_CNT_STEPS - STEP - 175000          :JMPN(outOfCountersStep)
```

Comment:
The worst case is 172.970. Don't you prefer, just in case, to go to 180k?

Reply:
I think 175K is enough given the 5% safe guard that we are applying